src/configure.rs

```diff
@@ -114,13 +105,13 @@ impl Config {
 impl Default for Config {
     fn default() -> Self {
         Self {
-            minimum_length: MINIMUM_LENGTH_DEFAULT,
-            vertex_spacing: VERTEX_SPACING_DEFAULT,
-            dummy_vertices: DUMMY_VERTICES_DEFAULT,
-            ranking_type: RANKING_TYPE_DEFAULT,
-            c_minimization: C_MINIMIZATION_DEFAULT,
-            transpose: TRANSPOSE_DEFAULT,
-            dummy_size: DUMMY_SIZE_DEFAULT,
+            minimum_length: 1,
+            vertex_spacing: 10,
+            dummy_vertices: true,
+            ranking_type: RankingType::MinimizeEdgeLength,
+            c_minimization: CrossingMinimization::Barycenter,
+            transpose: true,
+            dummy_size: 1.0,
```

Comment:
```suggestion
            minimum_length: MINIMUM_LENGTH_DEFAULT,
            vertex_spacing: VERTEX_SPACING_DEFAULT,
            dummy_vertices: DUMMY_VERTICES_DEFAULT,
            ranking_type: RANKING_TYPE_DEFAULT,
            c_minimization: C_MINIMIZATION_DEFAULT,
            transpose: TRANSPOSE_DEFAULT,
            dummy_size: DUMMY_SIZE_DEFAULT,
```
See the above comment.

Reply:
Done.